influxdb/client.py

```diff
@@ -195,21 +195,40 @@ def query(self,
               query,
               params={},
               expected_response_code=200,
+              time_precision='s',
               database=None,
-              raw=False):
+              raw=False,
+              chunked=False):
         """
         Query data
 
         :param params: Additional parameters to be passed to requests.
+        :param time_precision: [Optional, default 's'] Either 's', 'm', 'ms'
+            or 'u'.
         :param database: Database to query, default to None.
         :param expected_response_code: Expected response code. Defaults to 200.
         :param raw: Wether or not to return the raw influxdb response.
+        :param chunked: [Optional, default=False] True if the data shall be
+            retrieved in chunks, False otherwise.
+
         """
 
+        if time_precision not in ['s', 'm', 'ms', 'u']:
+            raise Exception(
+                "Invalid time precision is given. (use 's', 'm', 'ms' or 'u')")
+
         params['q'] = query
         if database:
             params['db'] = database
 
+        if time_precision:
```

Comment:
This condition is useless if we default the time precision to 's'. I would prefer to default the precision to None so that we use the InfluxDB default. I'll fix it myself when I merge.